tests/integration/test_remoted/test_agent_communication/test_request_agent_info.py

```diff
@@ -19,15 +19,11 @@
 configurations_path = os.path.join(test_data_path, 'wazuh_request_agent_info.yaml')
 
 parameters = [
-    {'PROTOCOL': 'udp,tcp'},
-    {'PROTOCOL': 'tcp'},
-    {'PROTOCOL': 'udp'},
+    {'PROTOCOL': 'udp,tcp'}
```

Comment:
I think that we should keep all the possible combinations of protocols that we have now since Wazuh 4.2. Before this version, the manager only accepted connections using only one protocol, but now it accepts both, TCP and UDP simultaneously.
This is just a suggestion so we can discuss this in our daily meeting to make a general decision for all the test that have single protocols and combination of them.